keps/sig-network/4962-network-topology-standard/README.md

```diff
@@ -154,13 +154,20 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 
 ## Summary
 
-This document proposes a standard for declaring switch network topology in Kubernetes clusters, representing the hierarchy of nodes, switches, and interconnects. In this context, a switch can refer to a physical network device or a collection of such devices with close proximity and functionality.
+This document proposes a standard for declaring switch network topology in Kubernetes clusters,
```

Comment:
I would remove the reference to switch network topology as has connotations, I think we are talking here about "Hierarchical Network Architectures", as we have hierarchy of "networks" with different levels, and the components at the same level should expect the same network properties